src/main/java/picard/vcf/CollectVariantCallingMetrics.java

```diff
@@ -293,8 +312,16 @@ public void calculateDerivedFields() {
             super.calculateDerivedFields();
             // Divide by zero should be OK -- NaN should get propagated to metrics file.
```

Comment:
That's not what I meant. You might have NaNs in the derived metrics from which you try to infer the private metrics. You need to be able to work-around that.